### DISCUSSION TITLE
Changing Title on Page

### DISCUSSION THREAD

**Discussion by leahmayschneider** [serenity-is/Serenity#5309] [Help]
I'm looking to change the title of the page. See image below (Funds should be Test)/ It doesn't seem to do anything to change the title part of the cshtml file (eg ViewData["Title"] = "Test";). I don't want to change the instance name. Is this possible?

![image](https://user-images.githubusercontent.com/8422795/102402913-789a3500-3fb3-11eb-89f2-8c25b8d8c568.png)

**Comment by VictorTomaili** [comment]
Use discussion for questions. 

You can override title on `grid.ts` file like this

```ts
public getInitialTitle() {
    return "My Title";
}
```
